CODEOWNERS

```diff
@@ -0,0 +1 @@
+@nestor-maintainers
```

Comment:
How about putting the file under the `.github` directory ?

Reply:
I prefer to keep it at the root for making it clear. Same for README and CONTRIBUTE